simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,33 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(binaryList: List[int]) -> List[int]:
+    lenLst = len(binaryList)
+    
+    if lenLst == 0:
+        return -1
+    
+    for digit in binaryList:
+        if digit != 0 and digit != 1:
+            return -1
+
+    digit = 0
+    listBinary = []
```

Comment:
listBinary? what is the point of revert the name? it make less sense and confusing. You can consider 2 options:

1/ binaryListIncrement (recommended)
2/ binaryListResult